src/Schema/AbstractColumn.php

```diff
@@ -623,6 +623,26 @@ public function isReadonlySchema(): bool
         return $this->getAttributes()['readonlySchema'] ?? false;
     }
 
+    public function first(bool $first = true): self
+    {
+        return $this;
+    }
+
+    public function isFirst(): bool
+    {
+        return false;
+    }
+
+    public function after(string $column): self
+    {
+        return $this;
+    }
+
+    public function getAfter(): string
+    {
+        return '';
+    }
```

Comment:
This feature looks like MySQL specific. Do we really need to declare it in other drivers?
I think we don't.